packages/helm-charts/blockscout/scripts/grafana-hook.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+echo Creating Grafana tag
+export CELO_ENV=$CELO_ENV
+export SUFFIX=$SUFFIX
+
+GRAFANA_API_TOKEN=$(echo $GRAFANA_CLOUD_SECRET | jq -r '.grafana_api_token')
```

Comment:
We can just pass two secrets, that way we won't need to depend on an external dependency (jq).